### PR TITLE
Fixed bug with taking Max of negative data

### PR DESCRIPTION
**Pull request by rhp0**
Hi - great program, but MAX was giving LDBL_MIN (nearly  zero) for negative data

### CODE REVIEW

**Comment by simonccarter** [comment]
Thanks. This project really needs a small testing framework set-up. If you can suggest any frameworks or help out with that, that would be great.

**Comment by rhp0** [comment]
I'm a neophyte, I don't know anything about software testing. This is
actually the first time I have ever contributed to open source - had to
learn about git yesterday to do it.
I would think that if sta were easier to find on google you would get more
people like me testing it by using it.
I googled "command line statistics" and came upon "st" and then found it
too hard to install perl dependencies on my embedded linux target - (never
liked perl anyway). Then I tried to get "datastats", but sourceforge was
down, so I had to keep searching. Found a few others I didn't like, then
finally came on your elegant masterpiece.
So bottom line is it's hard to find, and deserves more attention. I only
found it because sourceforge was down.

Regards,
Bob

**Comment by simonccarter** [comment]
Not a problem. I've pushed in some changes that now include testing. Will try and get the word out a bit more.
